File: app/javascript/routes/experimental.ts
// Experimental routes - UI prototypes and design experiments
// These routes are loaded conditionally in development only
// See routes/index.ts for loading logic

import type { RouteRecordRaw } from 'vue-router'

const experimentalRoutes: RouteRecordRaw[] = [
  {
    path: '/login2',
    name: 'login2',
    component: () => import('@/pages/Login2.vue'),
    meta: { experimental: true },
  },
  {
    path: '/components/:componentId/editor2',
    name: 'component_editor2',
    component: () => import('@/pages/components/Editor2DemoPage.vue'),
    meta: { experimental: true },
  },
]

export default experimentalRoutes

File: config/routes/experimental.rb
# frozen_string_literal: true

# Experimental routes - loaded only in development/test environments
# See config/routes.rb for conditional loading
#
# These routes are for UI experiments and prototypes that should NOT
# be available in production until they're production-ready.

# Login2 - Classified terminal aesthetic experiment (frontend-design skill)
get '/login2', to: 'public#show' # No auth required for demo

# RequirementEditor2 - Bootstrap 5 + reference slideover experiment
get '/components/:id/editor2', to: 'projects#index' # SPA shell for Vue Router
